client/ayon_hiero/api/plugin.py

```diff
@@ -593,31 +598,44 @@ def load(self):
         return track_item
 
 
-class Creator(LegacyCreator):
+class HiddenHieroCreator(HiddenCreator):
+    host_name = "hiero"
```

Comment:
Why has `host_name` attribute?

Reply:
It is not required, makes sense only if you would somehow register the create plugins path for other hosts.

Reply:
Got it, thanks for the explanation.